From 8ff345b0531b3692d59a090ac839080b44ab0641 Mon Sep 17 00:00:00 2001
From: Jared Morrow <jm@joyent.com>
Date: Wed, 20 Dec 2017 18:23:16 +0000
Subject: [PATCH] MANTA-3473 Update Muppet to node v4 Reviewed by: Cody Peter
 Mello <cody.mello@joyent.com> Approved by: Cody Peter Mello
 <cody.mello@joyent.com>

---
 targets.json.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/targets.json.in b/targets.json.in
index f2009ce..ebf1a5c 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1113,12 +1113,12 @@ cat <<EOF
     "build_platform": "20141030T081701Z",
     "image_name": "manta-loadbalancer",
     "image_description": "Manta loadbalancer",
-    "image_uuid": "de411e86-548d-11e4-a4b7-3bb60478632a",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
       "py27-curses",
-      "openssl-1.0.1u",
-      "stud-0.3p53nb2",
-      "zookeeper-client-3.4.6"
+      "openssl-1.0.2k",
+      "stud-0.3p53nb5",
+      "libzookeeper-3.4.6"
     ],
     "repos": [
       {"url": "git@github.com:joyent/muppet.git"}
-- 
2.21.0

